.changeset/proud-lions-whisper.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/ui5-application-writer': patch
```

Comment:
Add freestyle writer here or I suppose it will be bumped anyway as app writer is a dependency?